cmd/feed-ingress/main.go

```diff
@@ -167,6 +174,11 @@ func init() {
 			" '/myhost/myapp/health/'. Can be overridden with the sky.uk/exact-path annotation per ingress")
 	flag.IntVar(&healthPort, "health-port", defaultHealthPort,
 		"Port for checking the health of the ingress controller on /health. Also provides /debug/pprof.")
+	flag.StringVar(&ingressName, "ingress-name", defaultIngressName,
+		"Which ingress this instance is for.  This is used to filter ingresses so that only those with this"+
+			"ingress name will be updated.")
+	flag.StringVar(&namespaceSelector, namespaceSelectorFlag, defaultNamespaceSelector,
+		"Only consider ingresses within namespaces having labels matching this selector (e.g. app=loadtest).")
```

Comment:
I'm not sure. Since this is an open-source project I think we should stick with the Kubernetes notion of labels/annotations, which says:

> Labels can be used to select objects and to find collections of objects that satisfy certain conditions. In contrast, annotations are not used to identify and select objects.

https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/#attaching-metadata-to-objects

Reply:
Perhaps it would even be beter to use the namespace.labels["team"]?
`group: DL-CIS SE Platform Technology - Pom` vs `team: cats`